### PR TITLE
Replaced iconRight with iconPosition

### PR DESCRIPTION
**Pull request by hetanthakkar1**
Solved issue #2591

### CODE REVIEW

File: src/buttons/Button.tsx [hunk 1]
```diff
@@ -84,6 +85,7 @@ const Button: RneFunctionComponent<ButtonProps> = (props) => {
     linearGradientProps,
     ViewComponent = View,
     theme,
+    iconPosition,
```
Comment (arpitBhalla) [inline src/buttons/Button.tsx:88]: iconPosition='left'

File: src/buttons/Button.tsx [hunk 2]
```diff
@@ -159,6 +167,10 @@ const Button: RneFunctionComponent<ButtonProps> = (props) => {
           {...linearGradientProps}
           style={StyleSheet.flatten([
             styles.button,
+            styles.buttonOrientation,
+            {
+              flexDirection: iconPosition ? positionStyle[iconPosition] : 'row',
```
Comment (arpitBhalla) [inline src/buttons/Button.tsx:172]: positionStyle[iconPosition] || 'row',

**Comment by hetanthakkar1** [comment]
@flyingcircle @arpitBhalla I made the default to left

**Comment by arpitBhalla** [comment]
@flyingcircle This seems good to me

**Comment by hetanthakkar1** [comment]
@flyingcircle Please review

**Review by khushal87 — COMMENTED** [review]
Looks good to me @flyingcircle 💯

**Pull request merged**